src/main/java/org/broadinstitute/hellbender/tools/spark/sv/AlignAssembledContigsSpark.java

```diff
@@ -84,4 +76,24 @@ protected void runTool(final JavaSparkContext ctx) {
         assembledBreakpoints.saveAsTextFile(output);
     }
 
+    static JavaPairRDD<String, ContigsCollection> getContigsCollectionKeyedByBreakpointId(final JavaSparkContext ctx, final String inputPath) {
```

Comment:
Changed name to `loadContigsCollectionKeyedByAssemblyId` to make it clear that we are loading from disk, and added a comment.